cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,11 @@
+export class Address {
+  private checkoutSelector: string;
```

Comment:
the element should be called after what it is, a button in this case "checkoutButtonSelector" for example